src/DGmethods/DGmodel.jl

```diff
@@ -8,124 +8,110 @@ end
 
 function (dg::DGModel)(dQdt, Q, param, t; increment=false)
   bl = dg.balancelaw
+  DFloat = eltype(Q)
   device = typeof(Q.Q) <: Array ? CPU() : CUDA()
 
   grid = dg.grid
-  topology = grid.topology
-
-  dim = dimensionality(grid)
-  N = polynomialorder(grid)
-  Nq = N + 1
-  Nqk = dim == 2 ? 1 : Nq
-  Nfp = Nq * Nqk
-  nrealelem = length(topology.realelems)
-
-  Qvisc = param.diff
-  auxstate = param.aux
+  Ω    = grid.topology.realelems
+  nΩ   = length(Ω)
+  Nᵈ   = dimensionality(grid)
+  N    = polynomialorder(grid)
+  Nq   = N + 1
+  Nqk  = Nᵈ == 2 ? 1 : Nq
+  Nfp  = Nq * Nqk
+  Np   = dofs_per_element(grid)
+
+  σ  = param.diff
+  α  = param.aux
+  nσ = num_diffusive(bl, DFloat)
 
-  DFloat = eltype(Q)
-  nviscstate = num_diffusive(bl, DFloat)
-
-  lgl_weights_vec = grid.ω
-  Dmat = grid.D
   vgeo = grid.vgeo
   sgeo = grid.sgeo
-  vmapM = grid.vmapM
-  vmapP = grid.vmapP
-  elemtobndy = grid.elemtobndy
-  polyorder = polynomialorder(dg.grid)
-
-  Np = dofs_per_element(grid)
-
-  if hasmethod(update_aux!, Tuple{typeof(dg), typeof(bl), typeof(Q),
-                                  typeof(auxstate), typeof(t)})
-    update_aux!(dg, bl, Q, auxstate, t)
+  ω    = grid.ω
+  D    = grid.D
+  ι⁻   = grid.vmapM
```

Comment:
Personally I like `vmap*`, (especially since it's hard to see what character exactly `ι` is in my browser font)

Reply:
we can replace \iota. I used simple roman i in our implementation of the textbook code. main point is that it is a collection of indices for the GL points, so the name/symbol should give some reference to that

Reply:
@jkozdon bump

Reply:
I not sure what you're asking. vmapM = volume map to the minus side, that seems pretty clear to me.

Reply:
Also, if folks want to alias this to `ι⁻` in the kernels that's fine. I'd personally keep a more descriptive name in the type. But even that would be fine to change if it doesn't make sense to you what this is.

Reply:
oh so that's vmap meant! All summer I thought it meant "vector map" ... I like the symbol with iotas, but using vmap makes it easier to follow for anyone who has read the DG book

Reply:
`vmap` = volume map
`fmap` = face map (sometimes just `map` in the book codes)

Reply:
okay, maybe for the grid struct we can expand out to `volume_map` and `face_map` and use `vmap` and `fmap` in the kernels themselves. I can't think of a good symbol that encompasses both

Reply:
actually decided to just go with `M` for map here since we are only using the volume maps in these kernels